### PR TITLE
Update to allow for sendEmail rpm from copr

### PR DESCRIPTION
**Pull request by luigizuccarelli**
@danbev @grdryn - Please review

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ FROM centos:centos7
 
 EXPOSE 8080
 
+ADD sendEail-epel-7.repo /etc/yum.repos.d/
```
Comment (grdryn) [inline Dockerfile]: The filename is wrong here :)

Comment (grdryn) [inline Dockerfile]: @vibe13 Your productized builds should run without this line.

**Comment by grdryn** [comment]
@luigizuccarelli looks fine to me. I didn't try it out though, I'll leave that up to you or someone else

**Comment by luigizuccarelli** [comment]
@grdryn Installed the rpm on my CentOS local vm and also used the sendEmail to verify it could send mail

**Pull request merged**